### ISSUE TITLE
License is missing for vscode-account

### ISSUE THREAD

**Issue by spoenemann** [theia-ide/vscode-builtin-extensions#39]
The extension `vscode.vscode-account` is missing a LICENSE. All extensions must have a license according to the new terms set by the Eclipse Foundation.

@marcdumais-work could you republish that extension with a license?

**Comment by marcdumais-work** [comment]
Hi @spoenemann ,

I think that extension was only published initially, before we refined the process to add the license and other things. Do you know that it's useful outside of the context of VS Code? Should we publish it, going forward? If we do so I expect that the License will be present on newly published versions.

**Comment by marcdumais-work** [comment]
I just checked, and `vscode-account` is not longer part of the `vscode` repo. I see it in 1..45.0 but not in 1.46.0 or later. Maybe the best option is to unpublish that extension from `open-vsx.org`?

**Comment by marcdumais-work** [comment]
```
commit 9559d186d43e430878ead94905810b6e0f23a5f8
Author: Rachel Macfarlane <ramacfar@microsoft.com>
Date:   Wed May 13 09:41:29 2020 -0700

    Rename vscode-account extension to microsoft-authentication
```

**Comment by spoenemann** [comment]
Ok, then we'll just delete it.

**Comment by marcdumais-work** [comment]
Was that the only builtin without a license you found?

**Comment by spoenemann** [comment]
There were a few others, but for those newer versions with a license are already available.

**Comment by marcdumais-work** [comment]
Were they that same version or various versions? If the former, we could I think have the CI re-publish that specific version. It may be necessary to delete the original extensions though, before re-publishing. Let me know if you think such a thing is necessary.

**Comment by spoenemann** [comment]
Two versions:

* vscode.debug-server-ready 1.39.1
* vscode.git-ui 1.39.1

and

* vscode.debug-server-ready 1.44.2
* vscode.git-ui 1.44.2
* vscode.github-authentication 1.44.2
* vscode.vscode-account 1.44.2

**Comment by spoenemann** [comment]
BTW @marcdumais-work could you sign the new Open VSX Publisher Agreement? This is important in order to retain the extensions you have published.

See step 2 in https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by marcdumais-work** [comment]
> BTW @marcdumais-work could you sign the new Open VSX Publisher Agreement? This is important in order to retain the extensions you have published.
> 
> See step 2 in https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

What I have published to the registry under my own name was for testing and to start to use the registry. I think it's ok to remove it all. Instead I plan to add those extensions that are missing and I think are still interesting by having them built by the automated service (i.e. PR to https://github.com/open-vsx/publish-extensions).

**Comment by spoenemann** [comment]
@marcdumais-work note that some of the extensions you published for testing seem to me widely used:

 * https://open-vsx.org/extension/jebbs/plantuml - 6.6K downloads
 * https://open-vsx.org/extension/eclipse-cdt/cdt-gdb-vscode - 1.7K downloads

It would be bad to delete these as many users might depend on them. Continuing to publish the third-party extensions via the automated service (@open-vsx) is a good approach, but we would need new versions to be published very soon so the extensions are kept available. Would you have time to create PRs this week?

**Comment by spoenemann** [comment]
Another very important extension is this, for which you provided the first versions:

 * https://open-vsx.org/extension/llvm-vs-code-extensions/vscode-clangd - 21K downloads

The latest versions have been published by @planger, though. @planger could you sign the new Publisher Agreement as well?

**Comment by planger** [comment]
Sure, I signed it now. Thanks for the notification!

**Comment by marcdumais-work** [comment]
Thanks @planger.

Miro, I will open a PR ASAP for the PlantUML extension. However I am not sure the open-vsx build service will be flexible enough to handle this one, as there is "some assembly required". We can discuss on the PR.

**Comment by marcdumais-work** [comment]
the discussion diverged a bit, but to conclude:

TL;DR: a license is automatically added whenever a builtin is build using this repo. It has been the case for a long while - if we rebuild/publish older versions of the vscode builtins using the latest master of this repo, they all will have the proper license included.
